go/enclave/storage/init/migration/db_migration.go

```diff
@@ -4,6 +4,7 @@ import (
 	"database/sql"
```

Comment:
I don't understand the reason for the changes in this file? Since the 002 file was removed

Reply:
It clears up comments and a variable name that threw me off the first time I was reading the code, and adds more info to the error. 

Made those changes while debugging the code initially, and felt like they would be good to keep.
Likely it would be good to keep the comment and the error, but perhaps change back the var name if you prefer ? 
What do you think ?